doc/api/fs.md

```diff
@@ -1789,6 +1789,11 @@ added:
     filename passed to the listener. **Default:** `'utf8'`.
   * `signal` {AbortSignal} An {AbortSignal} used to signal when the watcher
     should stop.
+  * `maxQueue` {number} Specifies the number of events to queue between iterations
+    of the {AsyncIterator} returned. **Default:** `2048`.
+  * `overflow` {string} Either `'swallow'` or `'throw'` when there are more events to be
```

Comment:
I think 'ignore' and 'error' may be more appropriate terms than 'swallow' and 'throw'. 

At least ignore over swallow. I'd be okay with throw or error. 

Unless did you get this from something else? I'm okay if this is the status quo too

Reply:
I have no preference either way. I took the terms from what @joyeecheung suggested when suggesting the options. I think `’swallow’` makes sense as it’s events that are swallowed. `’ignore’` on the other hand suggests an error state that is ignored. But that’s really splitting the hairs on my bald head :wink:

Reply:
`ignore` and `error` SGTM too (ignore is also used in child processes for stdio so also has a bit more consistency)

Reply:
Addressed in 1bf75ba001a7f9e196329fa3a109ff27cf85c36a